packages/dashmate/configs/system/mainnet.js

```diff
@@ -16,7 +16,7 @@ const mainnetConfig = lodashMerge({}, baseConfig, {
   },
   core: {
     docker: {
-      image: 'dashpay/dashd:18.2.2',
+      image: 'dashpay/dashd:19.0.0',
```

Comment:
Let's go straight to `19.1.0`?

Reply:
Agree

Reply:
along with the main network, devnet and testnet were updated